docs/usage_guide.rst

```diff
@@ -1326,15 +1343,15 @@ Timeout or price not found for timestamp
 Figure out which asset caused the price not found. Then check the historical price caches and make sure you have the historical price cache for that asset pair created. For example if you are creating a GBP profit/loss report and the asset is GNO then make sure to create the GNO -> GBP historical price cache. See :ref:`manage-historical-price-cache` on how to do it.
 
 
-Analytics
+Statistics
```

Comment:
Renaming `Analytics` to `Statistics` since the sidebar option is called `Statistics` atm

Reply:
Hmmm ... here we should be consistent between both the app and the website. In the site we call them "Analytics" or "Graphs": https://rotki.com/products/

![2022-05-26_22-26](https://user-images.githubusercontent.com/1658405/170574435-de0acb68-ced7-47d9-9a37-988fc18b977d.png)


cc @kelsos what do you think?

Reply:
As discussed at standup, will keep it `Statistics`.